configure.ac

```diff
@@ -0,0 +1,191 @@
+                [address to git repository of systemd, used in 'src' build mode]),
+        [RKT_STAGE1_SYSTEMD_SRC="$withval"],
+        [RKT_STAGE1_SYSTEMD_SRC="git://anongit.freedesktop.org/systemd/systemd"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_SRC)
+
+## STAGE1: Systemd version
+AC_ARG_WITH(stage1-systemd-version,
+        AS_HELP_STRING([--with-stage1-systemd-version=version],
+                [systemd version to build]),
+        [RKT_STAGE1_SYSTEMD_VER="$withval"],
+        [RKT_STAGE1_SYSTEMD_VER="v219"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_VER)
+
+## STAGE1: linker-defined custom STAGE1 image path, default is unset
+AC_ARG_WITH(stage1-image-path,
+        AS_HELP_STRING([--with-stage1-image-path],
+                [custom stage1 image path]),
+        [RKT_STAGE1_IMAGE="$withval"],
+        [RKT_STAGE1_IMAGE=""])
+
+RKT_STAGE1_IMAGE_FLAGS=
+# if stage1 image variable is set, add a linker flag to rkt defining the variable
+if test x"${RKT_STAGE1_IMAGE}" != x ; then
+   RKT_STAGE1_IMAGE_FLAGS="-ldflags \"-X main.defaultStage1Image '${RKT_STAGE1_IMAGE}'\""
+fi
+
+AC_SUBST(RKT_STAGE1_IMAGE_FLAGS)
+
+## STAGE1 D-Bus parameters
+AC_ARG_WITH(stage1-dbus,
+        AS_HELP_STRING([--with-stage1-dbus=no],
+                [turn of d-bus or not, 'yes' turns it on, 'no' turns it off]),
+        [RKT_STAGE1_DBUS="$withval"],
+        [RKT_STAGE1_DBUS="no"])
+
+AC_SUBST(RKT_STAGE1_DBUS)
+
+AC_ARG_WITH(stage1-dbus-src,
+        AS_HELP_STRING([--with-stage1-dbus-src=git-path],
+                [address to git repository of systemd, used in 'src' build mode]),
+        [RKT_STAGE1_DBUS_SRC="$withval"],
+        [RKT_STAGE1_DBUS_SRC="git://anongit.freedesktop.org/dbus/dbus"])
+ 
+AC_SUBST(RKT_STAGE1_DBUS_SRC)
+
+AC_ARG_WITH(stage1-dbus-version,
+        AS_HELP_STRING([--with-stage1-dbus-version=version],
+                [d-bus version to build]),
+        [RKT_STAGE1_DBUS_VER="$withval"],
+        [RKT_STAGE1_DBUS_VER="dbus-1.8.16"])
+
+AC_SUBST(RKT_STAGE1_DBUS_VER)
+
+if test x"${RKT_STAGE1_DBUS}" == xyes ; then
+
+AC_MSG_RESULT([stage1 will build D-Bus])
+
+AC_SEARCH_LIBS([_nss_files_gethostbyname_r], [nss_files], [],
+               [AC_MSG_ERROR([libnss files is needed to have D-Bus working properly. Install libc package])])
+AC_SUBST(NSS_FILES_LIB, [$ac_cv_search__nss_files_gethostbyname_r])
+fi
+
+
+# Checks for programs.
+AC_PROG_CC
+AC_PROG_INSTALL
+
+AC_CHECK_PROG([GIT],[git],[git])
+if test x"${GIT}" == x ; then
+    AC_MSG_ERROR([git not installed, apt-get install git-core])
+fi
+
+AC_CHECK_PROG([GOBINARY],[go],[go])
+if test x"${GOBINARY}" == x ; then
+    AC_MSG_ERROR([Go language binary not found.])
+fi
+
+AC_CHECK_PROG([BASH],[bash],[bash])
+if test x"${BASH}" == x ; then
+    AC_MSG_ERROR([bash not found])
+fi
+# Checks for libraries.
+
+# Checks for header files.
+
+# Checks for typedefs, structures, and compiler characteristics.
+AC_TYPE_PID_T
+
+# Checks for library functions.
+AC_FUNC_FORK
+AC_CHECK_FUNCS([fchdir getcwd])
```

Comment:
autoscan likes adding things like this but considering a relatively modern glibc is required checking for basic types (pid_t) or standard functions is pretty pointless.